### PR TITLE
XMLRPC interface to retrieve account data

### PR DESCRIPTION
**Pull request by dansan**
regarding https://github.com/spring/uberserver/issues/133

### CODE REVIEW

File: XmlRpcServer.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        password_enc = unicode(b64encode(LEGACY_HASH_FUNC(password).digest()))
+        client = _FakeClient(self._root)
+        self._proto.in_TESTLOGIN(client, unicode(username), password_enc)
+        logger.debug("client.reply: %s", client.reply)
+        if client.reply.startswith("TESTLOGINACCEPT %s" % username):
+            session = self._root.userdb.sessionmaker()
+            db_user = session.query(User).filter(User.username == username).first()
+            renames = list()
+            for rename in db_user.renames:
+                renames.append(rename.original)
```
Comment (abma) [inline XmlRpcServer.py]: with this line very likely each name is duplicate, its not needed.

**Comment by dansan** [comment]
I thought the simpler code would make up for the little non-optimization, but you are right and here (https://github.com/dansan/uberserver/commit/9e55292050227630143299207fc257880c001c31) is the better code.

**Comment by abma** [comment]
thanks!

**Comment by dansan** [comment]
Thank you :)
Please PM me, when this goes online.

**Pull request merged**